shaky/src/main/java/com/linkedin/android/shaky/FeedbackActivity.java

```diff
@@ -142,18 +150,23 @@ public void onReceive(Context context, Intent intent) {
             } else if (DrawFragment.ACTION_DRAWING_COMPLETE.equals(intent.getAction())) {
                 onBackPressed();
             } else if (FormFragment.ACTION_SUBMIT_FEEDBACK.equals(intent.getAction())) {
-                submitFeedbackIntent(intent.getStringExtra(FormFragment.EXTRA_USER_MESSAGE));
+                submitFeedbackIntent(intent.getStringExtra(FormFragment.EXTRA_USER_MESSAGE),
+                                     intent.getParcelableArrayListExtra(FormFragment.EXTRA_ATTACHMENTS));
             }
         }
     };
 
-    private void submitFeedbackIntent(@Nullable String userMessage) {
+    private void submitFeedbackIntent(@Nullable String userMessage,
+                                      @Nullable ArrayList<Parcelable> attachments) {
         Intent intent = new Intent(ACTION_END_FEEDBACK_FLOW);
 
         intent.putExtra(SCREENSHOT_URI, imageUri);
         intent.putExtra(TITLE, getString(getTitleResId(feedbackType)));
         intent.putExtra(MESSAGE, userMessage);
         intent.putExtra(USER_DATA, userData);
+        if (attachments != null && !attachments.isEmpty()) {
+            intent.putParcelableArrayListExtra(EXTRA_ATTACHMENTS, attachments);
```

Comment:
Any reason we need to define EXTRA_ATTACHMENTS in both the fragment and the activity?

Reply:
Well, I just follow the same logic. When delegate tried to get the bundled values, it can use the constants defined in the same class.

Reply:
Yea makes sense, i guess these are two different semantic bundles. Sounds good!